README.md

```diff
@@ -53,6 +64,24 @@ There are two main sources of corpora available for Icelandic:
 ## License
 This project is licensed under the Apache License 2.0 - see the [LICENSE](LICENSE) file for details.
 
+### Contributing
+If you would like to contribute to this repository please feel free to submit a pull request or open up an issue.
+
+#### Developer tools
+In order to avoid bugs and different formatting, please use the following tools when writing source code in the project.
+- `black` for formatting
+- `flake8` for linting
+- `pydocstyle` for warning when you are missing docstrings.
+- `mypy` for static type checking.
+
+These tools (excluding `black`) will be run when a pull request is submitted and the build will fail if these tools complain.
+
+#### Running the tests
+Just run:
+```
+pytest
+```
+
```

Comment:
Move to a separate CONTRIBUTING.md file and add a hyperlink underneath the Contributors section.
e.g.: https://github.com/atom/atom/blob/master/CONTRIBUTING.md